packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -72,6 +72,18 @@ function getContentBlocks( blocks, isContentBlock ) {
 	return result;
 }
 
+function getShowTabs( blockName, tabSettings = {} ) {
+	if ( tabSettings[ blockName ] !== undefined ) {
+		return tabSettings[ blockName ];
+	}
+
+	if ( tabSettings.default !== undefined ) {
+		return tabSettings.default;
+	}
```

Comment:
So at the moment you can set `default: false` and something like `'core/paragraph': true` to enable tabs only on one block. Or `default: true` and `'core/paragraph': false` to disable only one block.

That's pretty flexible, and honors the individual tab setting, which is nice.

The other option might be having 'default' as a separate setting entirely instead of a property, but I don't really have a strong preference either way. I wonder if anyone has ever made a block with the name 'default', as that may cause an issue. It would be going against the block naming guidelines, which recommend namespaced block names (my-plugin/default), so probably not something to worry about.

Reply:
Thanks. I'll be happy to trial the alternative if others have stronger opinions or different thoughts.